tika-core/src/main/java/org/apache/tika/language/detect/LanguageNames.java

```diff
@@ -42,13 +42,13 @@ public static String normalizeName(String languageTag) {
     }
 
     public static boolean isMacroLanguage(String languageTag) {
-        Locale locale = Locale.forLanguageTag(languageTag);
+        // Locale locale = Locale.forLanguageTag(languageTag); unused
         // TODO make it so.
         return false;
     }
 
     public static boolean hasMacroLanguage(String languageTag) {
-        Locale locale = Locale.forLanguageTag(languageTag);
+        // Locale locale = Locale.forLanguageTag(languageTag); // unused
```

Comment:
Why not just remove?

Reply:
there is a todo below, commented line may help later